docs/content/en/docs/feature-status/_index.md

```diff
@@ -106,7 +106,7 @@ Please note that the phases (Incubating, Alpha, Beta, and Stable) are applied to
 | GitHub & GitHub Enterprise SSO | Beta |
 | Google SSO | Incubating |
 | Data Store - Support GCP [Firestore](https://cloud.google.com/firestore) | Beta |
-| Data Store - Support [MongoDB](https://www.mongodb.com/) | Alpha ([deprecated](/blog/2021/04/01/migrate-control-plane-datastore-to-mysql/)) |
+| Data Store - Support [MongoDB](https://www.mongodb.com/) | Alpha ([deprecated](/blog/2021/04/01/migrate-control-plane-datastore-from-mongodb-to-mysql)) |
```

Comment:
I'm guessing the link path is still `migrate-control-plane-datastore-to-mysql` because it's based on the markdown file name.

Reply:
I used to think like you too 😂 But tested with the new link 😁

Reply:
Really? A little bit odd 🤔 
I don't know but someone may be subscribing to this blog and bookmark this page. We want to avoid the link change as much as possible. It's on you.

Reply:
You're right 👍 I unchanged the link to that post 🙏